_observing-your-data/alerting/index.md

```diff
@@ -9,17 +9,38 @@ redirect_from:
 ---
 
 # Alerting
-OpenSearch Dashboards
-{: .label .label-yellow :}
 
-You can use the Alerting plugin in OpenSearch Dashboards to monitor your data and create alert notifications that trigger when conditions occur in one or more indexes.
+To create an alert, you configure a _monitor_, which is a job that runs on a defined schedule and queries OpenSearch indexes; configure one or more _triggers_, which define the conditions that generate events; and configure _actions_, which is what happens after an alert is triggered.
```

Comment:
Are all three steps required or is something optional (like actions, for example)? I would make this a bulleted list and add whether the step is required/optional.

Reply:
@AWSHurneyt Please confirm my understanding. Bullets 1 and 2 are required. Bullet 3 is optional.